RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -24,7 +26,9 @@
     "drop recoPFClusters_*_*_*",
     "keep recoElectronSeeds_*_*_*",
     "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
+    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
+    'keep recoPhotons_gedPhotonsTmp_*_*',
```

Comment:
On 8/12/15 10:45 AM, mandrenguyen wrote:
> In RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py
> <https://github.com/cms-sw/cmssw/pull/10700#discussion_r36875840>:
> 
>> @@ -24,7 +26,9 @@
>>      "drop recoPFClusters_*_*_*",
>>      "keep recoElectronSeeds_*_*_*",
>>      "keep recoGsfElectrons_*_*_*",
>> -    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
>> +    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
>> +    'keep recoPhotons_gedPhotonsTmp_*_*',
> 
> @slava77 <https://github.com/slava77> "Intentional" is probably the
> wrong word, but up until now the PF in the heavyIon scenario only
> includes the sequences with 'Tmp' appended at the end. At the time this
> was set up, I understood that the rest of the PF sequence was to remove
> PU, which won't do anything in for the HI reco, as we only reconstruct a
> single PV.


No, the rest of the PF sequence leading to products without Tmp
is not about PU removal.
It updates the refs pointing to PFCandidates and
adds PF-specific info as isolation or updates momentum  based on PF
knowledge (for muons)


> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10700/files#r36875840>.
>